configurator/src/MasterConfigurationPage.ui

```diff
@@ -2,6 +2,14 @@
 <ui version="4.0">
  <class>MasterConfigurationPage</class>
  <widget class="QWidget" name="MasterConfigurationPage">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>592</width>
+    <height>523</height>
+   </rect>
+  </property>
```

Comment:
In general make sure to reset the `geometry` property of the top level widget whenever changing UI files so this environment-specific data (which may change on every edit depending on the developer) does not clutter the history ;-)

Reply:
How would I do that? Just remove those lines Qt Creator added? As I'm still very new to Qt I sadly have no idea. :-(

Reply:
When editing a UI file, simply select the top level widget and go to the properties table and look for the `geometry` property and click the reset button on the right. Afterwards `geometry` should not be marked as modified (bold) any longer.

Reply:
Done.